modules/nw-autoscaling-ingress-controller.adoc

```diff
@@ -1,18 +1,21 @@
 // Module included in the following assemblies:
 //
-// * networking/ingress-controller-configuration.adoc
+// * networking/ingress-operator.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="nw-autoscaling-ingress-controller_{context}"]
 = Autoscaling an Ingress Controller
 
-Automatically scale an Ingress Controller to dynamically meet routing performance or availability requirements such as the requirement to increase throughput. The following procedure provides an example for scaling up the default `IngressController`.
+Automatically scale an Ingress Controller to dynamically meet routing performance or availability requirements, such as the requirement to increase throughput.
```

Comment:
Not necessary, but just something to consider while you're editing this line - if it makes sense, maybe this abstract can start with "you can" for more of a user focus:
```suggestion
You can automatically scale an Ingress Controller to dynamically meet routing performance or availability requirements, such as the requirement to increase throughput.
```

Reply:
:heavy_check_mark: